demo/app/lib/views/settings.dart

```diff
@@ -14,9 +17,18 @@ class Settings extends StatefulWidget {
 
 class SettingsState extends State<Settings> {
   final TextEditingController _usernameController = TextEditingController();
+  final Future<SharedPreferences> prefs = SharedPreferences.getInstance();
+  bool? isSwitched;
```

Comment:
you can change `isSwitched = preferences.getBool('devmode');` to `isSwitched = preferences.getBool('devmode') ?? false;` and change `bool? isSwitched` to `bool isSwitched = false` to get rid of ugly  ` isSwitched!=null?isSwitched!:false,